package.json

```diff
@@ -8,15 +8,19 @@
     "type": "git",
     "url": "https://github.com/AtomLinter/linter-tslint.git"
   },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "coffeelint ./lib"
+  },
   "license": "MIT",
   "engines": {
     "atom": ">0.50.0"
   },
   "dependencies": {
-    "atom-package-deps": "^4.0.1",
+    "atom-package-deps": "4.0.1",
```

Comment:
I would highly recommend _against_ this, with GreenKeeper enabled on this repo and tests + CI builds enabled all versions will be tested for you. If one of your allowed versions breaks things you will get a PR telling you so, from there you can either pin the version or fix the thing that is broken.

If you pin it like this though you will get a PR for _every single update_, since it will be outside your allowed range.